articles/openshift/oshift-how-use-netpol.md

```diff
@@ -0,0 +1,134 @@
+---
+title: How to use NetworkPolicy objects to connect services between projects | UKCloud Ltd
+description: Shows how to create a basic NetworkPolicy object to connect services in two projects
+services: openshift
+author: Kieran O'Neill
+
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Use NetworkPolicy objects
+toc_fullpath: How To/oshift-how-use-netpol.md
+toc_mdlink: oshift-how-use-netpol.md
+---
+
+# How to use NetworkPolicy objects to connect services between projects
+
+## Overview
+
+In v3.11 clusters deployed from mid-February 2019, the default SDN plugin has been changed from ovs-multitenant to ovs-networkpolicy. This gives you the ability to create NetworkPolicy objects which allow granular control over the flow of communication between the pods, services and projects inside your cluster. 
+
+By default, we create a NetworkPolicy object, named allow-from-same-and-privileged-namespaces, in every project, including those created after the cluster is deployed. This object allows all pods to communicate with each other inside the same project and allow communication from all pods and services to the default and openshift-monitoring projects. This mirrors the default pod security provided with the ovs-multitenant plugin.
+
```

Comment:
```suggestion

## Verifying the cluster SDN

In order to verify if the cluster has the ovs-networkpolicy SDN you can use the following command:

```oc get clusternetwork```

This will show redhat/openshift-ovs-networkpolicy under the PLUGIN NAME. If this shows ovs-multitenant instead you and you wish to use NetworkPolicy objects you will need to raise a service request to look at the options for switching SDN.

```